forum/qa-include/qa-page.php

```diff
@@ -473,6 +473,7 @@ function qa_page_routing()
 			'users' => 'pages/users.php',
 			'users/blocked' => 'pages/users-blocked.php',
 			'users/special' => 'pages/users-special.php',
+			'blocked-users-list' => 'pages/pw-blocked-list.php'

```

Comment:
moim zdaniem powinien to robić plugin, zmiany w core q2a to ostateczność jeśli już nie da się uniknąć takich zmian.

Reply:
jw